.github/workflows/storybook_deploy_main.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy Main Storybook
+
+on:
+  schedule:
+    - cron: '0 7 * * *'
```

Comment:
@ekohl I don't think a PR merge triggers a push event.

Reply:
Per [docs](https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule), "Scheduled workflows run on the latest commit on the default or base branch."  I think that should be okay..?

Reply:
Pretty sure it does. See https://github.com/theforeman/puppet-katello_devel/actions which has builds for master, which were certainly not PRs.

Reply:
My original thought was to run the action when:
- `develop` is pushed to
- a storybook-related directory was updated

So this will avoid deploying on every push to develop, the majority of which are not storybook updates.

This seems possible with [the 'Example including paths' section here](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths)

You could test this out on your fork if you are not already.